protocol/messenger_handlers.go

```diff
@@ -0,0 +1,1607 @@
+		return m.handleCancelContactVerificationProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_SYNC_KEYPAIR:
+		return m.handleSyncKeypairProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_SYNC_SOCIAL_LINKS:
+		return m.handleSyncSocialLinksProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_SYNC_ENS_USERNAME_DETAIL:
+		return m.handleSyncEnsUsernameDetailProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_SYNC_ACTIVITY_CENTER_NOTIFICATIONS:
+		return m.handleSyncActivityCenterNotificationsProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_SYNC_ACTIVITY_CENTER_NOTIFICATION_STATE:
+		return m.handleSyncActivityCenterNotificationStateProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_COMMUNITY_EVENTS_MESSAGE:
+		return m.handleCommunityEventsMessageProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_COMMUNITY_EDIT_SHARED_ADDRESSES:
+		return m.handleCommunityEditSharedAddressesProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_SYNC_ACCOUNT_CUSTOMIZATION_COLOR:
+		return m.handleSyncAccountCustomizationColorProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_SYNC_ACCOUNTS_POSITIONS:
+		return m.handleSyncAccountsPositionsProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_COMMUNITY_EVENTS_MESSAGE_REJECTED:
+		return m.handleCommunityEventsMessageRejectedProtobuf(messageState, protoBytes, msg, filter)
+        
+           case protobuf.ApplicationMetadataMessage_COMMUNITY_PRIVILEGED_USER_SYNC_MESSAGE:
+		return m.handleCommunityPrivilegedUserSyncMessageProtobuf(messageState, protoBytes, msg, filter)
+        
+	default:
+		m.logger.Info("protobuf type not found", zap.String("type", string(msg.Type)))
+	}
+	return nil
+}
+
+
+func (m *Messenger) handleChatMessageProtobuf(messageState *ReceivedMessageState, protoBytes []byte, msg *v1protocol.StatusMessage, filter transport.Filter) error {
+	m.logger.Info("handling ChatMessage")
+	
+
+	
+	p := &protobuf.ChatMessage{}
+	err := proto.Unmarshal(protoBytes, p)
+	if err != nil {
+		return err
+	}
+
+	m.outputToCSV(msg.TransportMessage.Timestamp, msg.ID, messageState.CurrentMessageState.Contact.ID, filter.Topic, filter.ChatID, msg.Type, p)
+
+	return m.HandleChatMessage(messageState, p, msg)
+	
+}
+
+
+func (m *Messenger) handleContactUpdateProtobuf(messageState *ReceivedMessageState, protoBytes []byte, msg *v1protocol.StatusMessage, filter transport.Filter) error {
+	m.logger.Info("handling ContactUpdate")
+	
+
```

Comment:
This file won't be linted, since it's generated by a template, and it's hard to get both in a readable state (if I remove the extra lines the template becomes a bit unreadable), since I have to inline the for loops, hope that's ok

Reply:
I see now, thanks for the clarification.